kroxylicious/src/main/templates/metadata.properties

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+kroxylicious.version=${project.version}
```

Comment:
I wonder if we should include the API version here too, for completeness

Reply:
and include it in the `-V` output? something like:
```
java -jar kroxylicious/target/kroxylicious-0.2.0-SNAPSHOT.jar -V
kroxylicious 0.2.0-SNAPSHOT (api version: 0.2.0-SNAPSHOT)
```

Reply:
Maybe a good call yes.

Thinking about it, it should probably actually go in `META_INF` in the api jar.

Reply:
went with:

```
java -jar kroxylicious/target/kroxylicious-0.2.0-SNAPSHOT.jar -V
kroxylicious: 0.2.0-SNAPSHOT
kroxylicous apis: 0.2.0-SNAPSHOT
```